diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter2/comments/CommentedCodeTest.kt

```diff
@@ -55,9 +55,7 @@ class CommentedCodeTest : LintTestBase(::CommentsRule) {
            |    */
            |    return 0
            |}
-        """.trimMargin(),
-            LintError(4, 5, ruleId, "${COMMENTED_OUT_CODE.warnText()} println(a + 42)", false)
```

Comment:
Please change these example so that commented code triggers warning with new adjustments

Reply:
Fixed

Reply:
Sorry, I meant all examples :)
There is really not much sense to have so many positive tests